frontend/src/routes/session/logout/+server.ts

```diff
@@ -0,0 +1,15 @@
+import type { RequestHandler } from './$types';
+import { json } from '@sveltejs/kit';
+
+export const POST: RequestHandler = async function ({ cookies, fetch }) {
+    cookies.delete('refresh_token', {
```

Comment:
long term, we should send a request to backend to ideally post to idp to logout.

I skipped this for now because there are many ways an idp server can set this up and it gets complicated quickly